aiogram/dispatcher/handler/base.py

```diff
@@ -23,18 +31,20 @@ def __init__(self, event: T, **kwargs: Any) -> None:
         self.data: Dict[str, Any] = kwargs
 
     @property
-    def bot(self) -> Bot:
+    def bot(self) -> Optional[Bot]:
         if "bot" in self.data:
-            return self.data["bot"]
+            # TODO: remove cast
+            return cast(Bot, self.data["bot"])
         return Bot.get_current()
```

Comment:
Maybe it's a good idea to use get_current(no_error=False) there?

Reply:
I think yes